com.unity.ml-agents/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to
 ### Minor Changes
 #### com.unity.ml-agents / com.unity.ml-agents.extensions (C#)
 #### ml-agents / ml-agents-envs / gym-unity (Python)
+- Removed unused `reparameterization` parameter and member for `Policy` and `TorchPolicy` classes.
```

Comment:
We don't need to update the changelog as this parameter isn't part of any public-facing API.